site/src/components/form.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Form
+description: The form related components in the Atlas Design System
+template: standard
+---
+
+# Form
+
+Forms contain form control elements such as the following:
+
+- [`.button`](./button.md)
+- [`.checkbox`](./checkbox.md)
+- [`.help`](./help.md)
+- [`.input`](./input.md)
+- [`.label`](./label.md)
+- [`.radio`](./radio.md)
+- [`.select`](./select.md)
+- [`.textarea`](./textarea.md)
+- custom form elements
+
+## Form Field
+
+The `.field` component is a container for form control elements to provide consistency.
+It can be used to hold `.control` containers, which holds the individual control elements.
+
+`.field-label` is used on the label. Add `field-label-sm`/`field-label-lg` makes label a little bigger or smaller than the default.
+
+```html
+<div class="field">
+	<label class="field-label field-label-sm label" for="sample-form-input-sm"> Input </label>
+	<div class="control">
+		<input
+			id="sample-form-input-sm"
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
```

Comment:
Can this pr standardize where to insert the field note/help text/described-by?
Example:
https://primer.style/css/components/forms#form-group-validation

Reply:
I was going to save it for the form validation document, but I added a section about it. Once the form validation document is available, I can add a link to it.

Reply:
I think validation should be considered part of this document since most/all forms will use validation such as required fields.

Can we simplify the markup a bit? Maybe something like

```html
<div class="field">
     <label class="field-label">
     <div class="field-body">
         ... input and decorations ...
     </div>
</div>
```

Would be good to have a radio/checkbox example.

Reply:
Changed it to just checkbox as there are other input examples below.